### PR TITLE
Add async and defer attributes to bundles

### PR DESCRIPTION
**Pull request by thet**
Add ``load_async`` and ``load_defer`` attributes to resource registries bundle settings.
When set, ``<script>`` tags are rendered with ``async="async"`` resp. ``defer="defer"`` attributes.

Update:
- async/defer also for meta bundles (default.js, logged-in.js)
- updated combined bundles to make use of newest resource registry UI changes

Merge:
https://github.com/plone/Products.CMFPlone/pull/2644
https://github.com/plone/mockup/pull/879
https://github.com/plone/plone.app.upgrade/pull/186

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -12,6 +12,24 @@
...
+
+Breaking changes:
+
+- *add item here*
+
+New features:
+
+- Add ``load_async`` and ``load_defer`` attributes to resource registries bundle settings.
+  When set, ``<script>`` tags are rendered with ``async="async"`` resp. ``defer="defer"`` attributes.
+  [thet]
```
Comment (gforcada) [inline CHANGES.rst]: this should go to a file within ``news`` folder. See the comment on https://raw.githubusercontent.com/plone/Products.CMFPlone/master/CHANGES.rst for details.

**Comment by thet** [comment]
Tests are green. Next commit will only a news entry: https://jenkins.plone.org/job/pull-request-5.1/1659/

**Comment by jensens** [comment]
This goes to 5.1.x branch. I am sure we need the same for master too at the same time. mockup changes will go to 5.2 as well. Then we need some upgrade step valid from 5.2.a1 to next 5.2 as well.

**Comment by agitator** [comment]
default.js should also get the async attribute if enabled on the plone bundle

**Comment by thet** [comment]
@peter: done
@jensens started at: https://github.com/plone/Products.CMFPlone/pull/2649

**Comment by jensens** [comment]
@thet does the upgrade work from a 5.2a1 away?

**Pull request merged**